bigtable/client/mutations.h

```diff
@@ -62,29 +62,140 @@ constexpr std::int64_t ServerSetTimestamp() { return -1; }
- * endpoints of the time range.  The Cloud Bigtable server rejects invalid and
- * empty ranges, i.e., any range where the endpoint is smaller or equal than to
- * the initial endpoint unless either endpoint is 0.
+ * points of the time range.  The Cloud Bigtable server rejects invalid and
+ * empty ranges, i.e., any range where the end point is smaller or equal than to
+ * the initial end point unless either end point is 0.
+ *
+ * @tparam Rep1 a placeholder to match the Rep tparam for @p timestamp_begin
+ * type.The semantics of this template parameter are documented in
+ * std::chrono::duration<>` (in brief, the underlying arithmetic type
+ * used to store the number of ticks), for our purposes it is simply a
+ * formal parameter.
+ *
+ * @tparam Rep2 similar formal parameter for the type of @p timestamp_end.
+ *
+ * @tparam Period1 a placeholder to match the Period tparam for @p
+ * timestamp_begin
+ * type.the semantics of this template parameter are documented in
+ * `std::chrono::duration<>` (in brief, the length of the tick in seconds,
+ * expressed as a `std::ratio<>`), for our purposes it is simply a formal
+ * parameter.
+ *
+ * @tparam Period2 similar formal parameter for the type of @p timestamp_end.
  */
-
-/// Delete only within the timestamp range provided.
+template <typename Rep1, typename Period1, typename Rep2, typename Period2>
 Mutation DeleteFromColumn(std::string family, std::string column,
-                          std::int64_t timestamp_begin,
-                          std::int64_t timestamp_end);
+                          std::chrono::duration<Rep1, Period1> timestamp_begin,
+                          std::chrono::duration<Rep2, Period2> timestamp_end) {
+  Mutation m;
+  using namespace std::chrono;
+  auto& d = *m.op.mutable_delete_from_column();
+  d.set_family_name(std::move(family));
+  d.set_column_qualifier(std::move(column));
+  d.mutable_time_range()->set_start_timestamp_micros(
+      duration_cast<microseconds>(timestamp_begin).count());
+  d.mutable_time_range()->set_end_timestamp_micros(
+      duration_cast<microseconds>(timestamp_end).count());
+  return m;
+}
+
+//@{
+/**
+ * @name The following functions create a mutation that deletes all the
+ * cells in the given column family and column, starting from and
+ * including, @a timestamp_begin.
+ *
+ * The function accepts any instantiation of `std::chrono::duration<>` for the
+ * @p timestamp_begin For example:
+ *
+ * @code
+ * using namespace std::chrono_literals; // C++14
+ * bigtable::DeleteFromColumn("fam", "col", 10us)
+ * @endcode
+ *
+ * @tparam Rep1 a placeholder to match the Rep tparam for @p timestamp_begin
+ * type.The semantics of this template parameter are documented in
+ * `std::chrono::duration<>` (in brief, the underlying arithmetic type
+ * used to store the number of ticks), for our purposes it is simply a
+ * formal parameter.
+ *
+ * @tparam Period1 a placeholder to match the Period tparam for @p
+ * timestamp_begin
```

Comment:
and here.  Also missing space after the `.`.

Reply:
After indention of  doxygen, I use clang-format-4.0 for formatting, before commit.
so some how, clang-format-4.0 changed the indentation again.
will check it